### ISSUE TITLE
When a pearled alt is re-pearled, it can kick for too many alts incorrectly.

### ISSUE THREAD

**Issue by GavJenks** [CivClassic/ExilePearl#36]
I don't know if this is EP or altmanager, will submit to both.

Reproduce: go on a server with, say, a limit of 2 alts pearled before alt banning. Pearl one of them. Summon and re-pearl him (Still only 1 pearled, not 2). Will get kicked and alt banned incorrectly.

**Comment by Lazersmoke** [comment]
Could you do this on the test server and let me know when you get it into this state? I want to see what the SQL tables look like when this happens. Could be either plugin, but AltManager looks good so it's likely ExilePearl.

**Comment by GavJenks** [comment]
Ok is in that state, gav is altbanned

**Comment by Lazersmoke** [comment]
Well the pearled alts cap on the test server is 1 so L.

**Comment by GavJenks** [comment]
Yeah in that case, it is a bug that I DON'T get alt banned in the initial pearling, heh.
